crypto/ec/ec_ameth.c

```diff
@@ -640,6 +640,27 @@ static int ec_pkey_import_from(const OSSL_PARAM params[], void *vpctx)
     return 1;
 }
 
+static int ec_pkey_copy(EVP_PKEY *to, EVP_PKEY *from)
+{
+    EC_KEY *eckey = from->pkey.ec;
+    EC_KEY *dupkey = NULL;
+    int ret;
+
+    if (eckey != NULL) {
+        dupkey = EC_KEY_dup(eckey);
```

Comment:
NOTE that this dup uses the libctx and propq of the eckey...

Reply:
Same applies to other key types..

Reply:
Yep, I do not see it as a problem.